### PR TITLE
Fix css syntax - !important #6146

### LINKED ISSUE #6146: Fix css syntax - !important
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file

* Line 97 in plots2/app/views/notes/_notes.html.erb
* Line 109,132,133 in plots2/app/views/tag/show/_header.html.erb

```diff
- padding: 0px; !important
+ padding: 0px !important;
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by nadchif**
Fixes #6146

### CODE REVIEW

File: app/views/notes/_notes.html.erb [hunk 1]
```diff
@@ -94,7 +94,7 @@
   border-color: #D3D3D3;
   border-radius: 6px;
   margin:1em 0em 1em 0em;
-  padding: 0px; !important
+  padding: 0px !important
```
Comment (CleverFool77) [inline app/views/notes/_notes.html.erb]:
```suggestion
  padding: 0px !important ;
```

File: app/views/tag/show/_header.html.erb [hunk 2]
```diff
@@ -106,7 +106,7 @@
 
 .im {
   position: relative;
-  padding: 0px; !important
+  padding: 0px !important
```
Comment (CleverFool77) [inline app/views/tag/show/_header.html.erb]:
```suggestion
  padding: 0px !important ;
```

File: app/views/tag/show/_header.html.erb [hunk 3]
```diff
@@ -129,8 +129,8 @@
 }
 
 #wiki-content {
-  padding: 0px; !important
-  margin: 0px; !important
+  padding: 0px !important
```
Comment (CleverFool77) [inline app/views/tag/show/_header.html.erb]:
```suggestion
  padding: 0px !important ;
```

File: app/views/tag/show/_header.html.erb [hunk 4]
```diff
@@ -129,8 +129,8 @@
...
+  margin: 0px !important
```
Comment (CleverFool77) [inline app/views/tag/show/_header.html.erb]:
```suggestion
  margin: 0px !important ;
```

**Comment by nadchif** [comment]
> Hi @nadchif Great Start !! 
> You forgot the semicolon in syntax. I have suggested it.
> Just update the changes and it would be done.
> Thanks !!

Thank you for bringing that to my attention. So sorry it had slipped past me :/ I have corrected it now

**Comment by Anupam-dagar** [comment]
Hey @nadchif, thanks for the contribution but the semicolon is still missing. Add the semicolon as suggested by @CleverFool77 in her review. It should be after every line of css.

**Comment by CleverFool77** [comment]
Hi @nadchif As already mentioned above, You have added semicolon in one line only, three lines are still left. 
Thanks !!

**Comment by nadchif** [comment]
Thank you @Anupam-dagar and @CleverFool77 . I feel so silly about that. I have reviewed it again :) this has been quite

**Comment by jywarren** [comment]
Great!!

**Comment by jywarren** [comment]
Thank you so much, @nadchif !!!

**Pull request merged**